pkg/ui/workspaces/cluster-ui/src/databaseDetailsV2/types.ts

```diff
@@ -8,6 +8,9 @@ import { NodeID } from "src/types/clusterTypes";
 
 export type TableRow = TableMetadata & {
   qualifiedNameWithSchema: string;
-  nodesByRegion: Record<string, NodeID[]>;
+  nodesByRegion: {
+    isLoading: boolean;
```

Comment:
There's no standard abstraction available around an "async loaded value"? Seems surprising to me to pepper the code with `isLoading` booleans.

Reply:
There isn't one today, no. swr explicitly provides the loading/validating state separately from the resolved promise value. I'm also treating this as more of a special case since the nodes information currently loads separately from the rest of the table metadata -- we could think about putting this as part of the job so we don't have to go through the nodes api like we do today.